CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.9.79](https://github.com/brave/brave-browser/releases/tag/v1.9.79)
+
+- Implemented import/export db options for rewards. ([#9766](https://github.com/brave/brave-browser/issues/))
+- Added settings for `Top sites suggestions` and `Brave suggested sites`. ([#9782](https://github.com/brave/brave-browser/issues/))
+- Added `Estimated Bandwidth Savings` stats on new tab page. ([#8845](https://github.com/brave/brave-browser/issues/))
```

Comment:
Same as the above, should be using " rather than ` as the above formats the text and historically we don't do that in any of the changelogs. Examples:

* Added `Estimated Bandwidth Savings` <-- formatted
* Added "Estimated Bandwidth Savings" <-- not formatted

Looks like GH issue # at the end of the link is missing which breaks the link/formatting: Example:

Current --> `([#8845](https://github.com/brave/brave-browser/issues/))`
Should be --> `([#8845](https://github.com/brave/brave-browser/issues/8845))`